src/app/blog-page/blog-detail/blog-detail.component.ts

```diff
@@ -386,28 +385,23 @@ export class BlogDetailComponent implements OnInit, OnDestroy {
 
         this.backendApi
           .SubmitPost(
-            this.globalVars.localNode,
             this.globalVars.loggedInUser?.PublicKeyBase58Check,
             this.currentPost.PostHashHex /*PostHashHexToModify*/,
             "" /*ParentPostHashHex*/,
-            "" /*Title*/,
             {
               Body: this.currentPost.Body,
               ImageURLs: this.currentPost.ImageURLs,
+              VideoURLs: [],
             } /*BodyObj*/,
             "",
             this.currentPost.PostExtraData,
-            "" /*Sub*/,
-            isHidden /*IsHidden*/,
-            this.globalVars.feeRateDeSoPerKB * 1e9 /*feeRateNanosPerKB*/
```

Comment:
I had to remove this field when was merging with the latest `main`. Please let me know if it's wrong

Reply:
We can sync on this. We will need to test on prod to make sure it still works